js_modules/dagit/packages/core/src/assets/AssetsCatalogTable.tsx

```diff
@@ -203,6 +226,68 @@ export const AssetsCatalogTable: React.FC<AssetCatalogTableProps> = ({
   );
 };
 
+const AssetGroupSuggest: React.FC<{
+  assets: Asset[];
+  value: AssetGroupSelector | null;
+  onChange: (g: AssetGroupSelector | null) => void;
+}> = ({assets, value, onChange}) => {
+  const assetGroups = React.useMemo(
+    () =>
+      uniqBy(
+        (assets || []).map(buildAssetGroupSelector).filter((a) => !!a) as AssetGroupSelector[],
+        (a) => JSON.stringify(a),
+      ).sort((a, b) => a.groupName.localeCompare(b.groupName)),
+    [assets],
+  );
+
+  const repoContextNeeded = React.useMemo(() => {
+    const result: {[groupName: string]: boolean} = {};
+    assetGroups.forEach(
+      (group) => (result[group.groupName] = group.groupName in result ? true : false),
+    );
+    return result;
+  }, [assetGroups]);
+
+  return (
+    <Suggest<AssetGroupSelector>
+      selectedItem={value}
+      items={assetGroups}
+      inputProps={{
+        style: {width: 220},
+        placeholder: 'Filter asset groups…',
+        rightElement: value ? (
+          <div style={{marginTop: 6.5, marginRight: 6}} onClick={() => onChange(null)}>
+            <Icon name="cancel" color={Colors.Gray400} />
+          </div>
```

Comment:
Make this a styled `button` instead of a div?

Reply:
Ahh good call, it looks like we have one on the logs filter we can reuse for this actually 🙌